src/transformers/blenderbot-90M-config.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I think you don't have to add a config json file to `transformers`, we can upload them on S3 and have it there :-)

Reply:
I will delete it after. Wanted to have your reviews before uploading it to S3